frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/StartHandler.java

```diff
@@ -188,19 +186,48 @@ public APIGatewayProxyResponseEvent handleRequest(
             LOG.info("previousSessionId: {}", previousSessionId);
             authSessionService.addOrUpdateSessionId(previousSessionId, session.getSessionId());
 
+            var clientSessionId =
+                    getHeaderValueFromHeaders(
+                            input.getHeaders(),
+                            CLIENT_SESSION_ID_HEADER,
+                            configurationService.getHeadersCaseInsensitive());
+            var txmaAuditHeader =
+                    getOptionalHeaderValueFromHeaders(
+                            input.getHeaders(), TXMA_AUDIT_ENCODED_HEADER, false);
+            String internalSubjectIdForAuditEvent = AuditService.UNKNOWN;
+            String internalCommonSubjectIdentifierForAuditEvent = AuditService.UNKNOWN;
+
+            var auditContext =
+                    new AuditContext(
+                            userContext.getClient().get().getClientID(),
+                            clientSessionId,
+                            session.getSessionId(),
+                            internalCommonSubjectIdentifierForAuditEvent,
+                            userContext
+                                    .getUserProfile()
+                                    .map(UserProfile::getEmail)
+                                    .orElse(AuditService.UNKNOWN),
+                            IpAddressHelper.extractIpAddress(input),
+                            AuditService.UNKNOWN,
+                            extractPersistentIdFromHeaders(input.getHeaders()),
+                            txmaAuditHeader);
+
+            boolean isBlockedForReauth = false;
+            if (configurationService.isAuthenticationAttemptsServiceEnabled() && reauthenticate) {
+                isBlockedForReauth =
```

Comment:
If a User is blocked from re-authenticating and they are attempting to re-authenticate why do we allow them to continue here?

Reply:
Makes sense.